frontend/src/pages/modelRegistrySettings/useRoleBindings.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { listRoleBindings } from '~/api';
+import { RoleBindingKind } from '~/k8sTypes';
+import useFetchState, { FetchState } from '~/utilities/useFetchState';
+
+const useRoleBindings = (): FetchState<RoleBindingKind[]> => {
+  const getRoleBindings = React.useCallback(
+    () =>
+      listRoleBindings('odh-model-registries', 'component=model-registry').catch((e) => {
```

Comment:
Instead of hard-coding the `odh-model-registries` namespace we should import and use `MODEL_REGISTRY_DEFAULT_NAMESPACE`.

Also, instead of hard-coding `component=model-registry` can we create a new `LABEL_SELECTOR_MODEL_REGISTRY` in `frontend/src/const.ts`? And as part of that can you create `KnownLabels.COMPONENT` in `frontend/src/k8sTypes.ts`?

Reply:
This could be a follow-up tech debt task if you can't get to it today.